cdap-app-fabric/src/main/java/co/cask/cdap/test/RuntimeStats.java

```diff
@@ -58,27 +58,40 @@ public static void resetAll() throws Exception {
     metricStore.deleteBefore(System.currentTimeMillis() / 1000);
   }
 
-  public static RuntimeMetrics getFlowletMetrics(String applicationId, String flowId, String flowletId) {
-    Id.Program id = Id.Program.from(Constants.DEFAULT_NAMESPACE, applicationId, ProgramType.FLOW, flowId);
+  public static RuntimeMetrics getFlowletMetrics(String namespace, String applicationId,
+                                                 String flowId, String flowletId) {
+    Id.Program id = Id.Program.from(namespace, applicationId, ProgramType.FLOW, flowId);
     return getMetrics(MetricsContexts.forFlowlet(id, flowletId),
                       MetricsConstants.FLOWLET_INPUT, MetricsConstants.FLOWLET_PROCESSED,
                       MetricsConstants.FLOWLET_EXCEPTIONS);
   }
 
-  public static RuntimeMetrics getProcedureMetrics(String applicationId, String procedureId) {
-    Id.Program id = Id.Program.from(Constants.DEFAULT_NAMESPACE, applicationId, ProgramType.PROCEDURE, procedureId);
+  public static RuntimeMetrics getFlowletMetrics(String applicationId, String flowId, String flowletId) {
+    return getFlowletMetrics(Constants.DEFAULT_NAMESPACE, applicationId, flowId, flowletId);
+  }
+
+  public static RuntimeMetrics getProcedureMetrics(String namespace, String applicationId, String procedureId) {
```

Comment:
At least in the program lifecycle APIs, I know there is a jira to drop support for procedures with v3 apis. we may want to remove this/ always use default namespace